libethereum/State.cpp

```diff
@@ -521,21 +531,11 @@ void State::rollback(size_t _savepoint)
 		case Change::Storage:
 			account.setStorage(change.key, change.value);
 			break;
-		case Change::StorageRoot:
-			account.setStorageRoot(change.value);
-			break;
-		case Change::Balance:
-			account.addBalance(0 - change.value);
-			break;
-		case Change::Nonce:
-			account.setNonce(change.value);
-			break;
-		case Change::Create:
-			m_cache.erase(change.address);
-			break;
-		case Change::Code:
-			account.setCode(std::move(change.oldCode));
-			break;
+		case Change::StorageRoot: account.setStorageRoot(change.value); break;
+		case Change::Balance: account.addBalance(0 - change.value); break;
+		case Change::Nonce: account.setNonce(change.value); break;
+		case Change::Create: m_cache.erase(change.address); break;
+		case Change::Code: account.setCode(std::move(change.oldCode)); break;
```

Comment:
this one is shorter but harder to read

Reply:
Yes, I agree. I will check if I can bring it back. But this is a question if we allow single-lines after `case:` in general. Sometimes it is useful, like

```
case A: return "A";
case B: return "B";
```

Reply:
I don't like these with return on the same line ^

Reply:
So fixed. I think we are consistent there with config:
```
AllowShortBlocksOnASingleLine: false
AllowShortCaseLabelsOnASingleLine: false
AllowShortIfStatementsOnASingleLine: false
```